x/ecocredit/server/core/msg_create_class.go

```diff
@@ -24,13 +24,27 @@ func (k Keeper) CreateClass(goCtx context.Context, req *core.MsgCreateClass) (*c
 		return nil, err
 	}
 
-	if err := k.assertCanCreateClass(sdkCtx, adminAddress); err != nil {
+	if err := k.assertCanCreateClass(goCtx, adminAddress); err != nil {
 		return nil, err
 	}
 
-	// TODO: remove params https://github.com/regen-network/regen-ledger/issues/729
-	var allowedFees sdk.Coins
-	k.paramsKeeper.Get(sdkCtx, core.KeyCreditClassFee, &allowedFees)
+	var fees *api.ClassFees
+	fees, err = k.stateStore.ClassFeesTable().Get(goCtx)
+	if err != nil {
+		if !ormerrors.NotFound.Is(err) {
+			return nil, err
+		}
+		fees = &api.ClassFees{}
+	}
+
+	allowedFees := make(sdk.Coins, 0, len(fees.Fees))
+	for _, coin := range fees.Fees {
+		amount, ok := sdk.NewIntFromString(coin.Amount)
+		if !ok {
+			return nil, sdkerrors.ErrInvalidType.Wrapf("credit class fee %s", coin.Amount)
+		}
+		allowedFees = append(allowedFees, sdk.NewCoin(coin.Denom, amount))
+	}
```

Comment:
i think this similar code snippet showed up in another PR - maybe we can make a util for it?